google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -0,0 +1,85 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.spi.v1
+  python:
+    package_name: google.cloud.speech.v1
+  go:
+    package_name: cloud.google.com/go/speech/apiv1
+  csharp:
+    package_name: Google.Cloud.Speech.V1
+  ruby:
+    package_name: Google::Cloud::Speech::V1
+  php:
+    package_name: Google\Cloud\Speech\V1
+  nodejs:
+    package_name: speech.v1
+    domain_layer_location: google-cloud
+license_header:
+  copyright_file: copyright-google.txt
+  license_file: license-header-apache-2.0.txt
+interfaces:
+- name: google.cloud.speech.v1.Speech
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
```

Comment:
I think @lukesneeringer determined that this should be `190000` based on https://github.com/googleapis/googleapis/pull/297